data-prepper-plugins/log-generator-source/src/test/java/com/amazon/dataprepper/plugins/source/loggenerator/CsvLogFakerTest.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.source.loggenerator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class CsvLogFakerTest {
+
+    @Test
+    public void when_CsvLogFakerVPCFlowLogs_then_hasCorrectFormat() {
+        CsvLogFaker objectUnderTest = new CsvLogFaker();
+
+        final String log = objectUnderTest.generateRandomStandardVPCFlowLog();
+        final int numberOfSpaces = log.length() - log.replace(" ","").length(); // count instances of space (" ")
```

Comment:
Nit: I think you could do something like `StringUtils.countMatches(log, " ");` instead